src/lib/onboard/runtime-provider/host-local-create-journal.ts

```
@@ -0,0 +1,616 @@
+      }
+    } else {
+      fs.renameSync(temporary, target);
+    }
+    fs.chmodSync(target, FILE_MODE);
+    fsyncDirectory(root);
+  } finally {
+    if (descriptor !== null) fs.closeSync(descriptor);
+    try {
+      fs.unlinkSync(temporary);
+    } catch (error) {
+      if ((error as NodeJS.ErrnoException).code !== "ENOENT") throw error;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)